quickstart/monitoring/monitoring-quickstart.ipynb

```diff
@@ -164,13 +165,19 @@
     "_PREDICTION_COLUMN_NAME = 'prediction'\n",
     "_PREDICTION_THRESHOLD = 0.6\n",
     "_PREDICTION_CLASSIFICATION_COLUMN_NAME = \"prediction_binary\"\n",
+    "# Replace this with a time to overwrite timestamp. Maximum allowed value is 15778476 Seconds i.e. 6 months\n",
```

Comment:
May be we should mention in terms of prod data. 
If _OVERWRITE_TIMESTAMP_SECS = 86400 => prod data is of last day
if _OVERWRITE_TIMESTAMP_SECS = (6*30) * 86400 => prod data is of last 6 month